debian/changelog

```diff
@@ -1,3 +1,11 @@
+argobots (1.1a1) unstable; urgency=medium
```

Comment:
Ditto on the `~` and you need to include the git hash here also:

```suggestion
argobots (1.1~a1-1.g7fd1987) unstable; urgency=medium
```

Reply:
ok